### PR TITLE
Obtain data from a schema

### PR DESCRIPTION
**Pull request by dalepotter**
~~ENSURE #57 IS MERGED BEFORE THIS ONE~~ < Now done

### CODE REVIEW

**Comment by hayfield** [comment]
@dalepotter What is the current status of this PR?

**Comment by dalepotter** [comment]
@hayfield Once the merge conflicts are resolved, and the `TODOs` reviewed, this one could be opened for review in it's current state.

**Comment by hayfield** [comment]
Not checked, but should ensure there is some sort of test to ensure element ordering is maintained.

File: iati/core/tests/test_schemas.py [hunk 1]
```diff
@@ -1,171 +1,630 @@
...
 import iati.core.codelists
+import iati.core.default
 import iati.core.exceptions
 import iati.core.schemas
 import iati.core.tests.utilities
 
 
-class TestSchemas(object):
-    """A container for tests relating to Schemas."""
+def default_activity_schema():
```
Comment (hayfield) [inline iati/core/tests/test_schemas.py]: There is a function under `default` that returns this.

File: iati/core/tests/test_schemas.py [hunk 2]
```diff
@@ -1,171 +1,630 @@
...
-        """
-        schema_name = iati.core.tests.utilities.SCHEMA_NAME_VALID
+    return iati.core.default.schema(schema_name)
 
-        return iati.core.Schema(name=schema_name)
 
-    def test_schema_default_attributes(self):
-        """Check a Schema's default attributes are correct."""
-        schema = iati.core.Schema()
+def default_organisation_schema():
```
Comment (hayfield) [inline iati/core/tests/test_schemas.py]: There is a function under `default` that returns this.

**Comment by hayfield** [comment]
Attempted to sort out the merge conflicts, though the number of distinct conflicts in `test_schema.py` make it feel dangerous - will leave this to you to sort out since you know the changes better than I do!

**Comment by hayfield** [comment]
It had been decided that this method of implementation will not be used due to the inherent level of complexity involved. An alternative method using XSLT will likely be utilised instead.

More detail is available in #303.

**Pull request closed without merging**